docker/dockerfile-build-ubuntu-rock

```diff
@@ -0,0 +1,48 @@
+# This Dockerfile provides a starting point for a ROCm installation of hipCaffe.

```

Comment:
not building hipCaffe

Reply:
Yep. We really need to fetch dockerfiles directly from rocJenkins and not in every repo...

Reply:
this project is called Tensile... not hipCaffe